src/test/scala/nodes/learning/LeastSquaresEstimatorSuite.scala

```diff
@@ -0,0 +1,92 @@
+package nodes.learning
+
+import breeze.linalg.{DenseVector, SparseVector}
+import org.apache.spark.SparkContext
+import org.scalatest.FunSuite
+import pipelines._
+import workflow.WorkflowUtils
+
+class LeastSquaresEstimatorSuite extends FunSuite with LocalSparkContext with Logging {
```

Comment:
1. Doesn't look like it in the existing "big n big d dense" test case I have, should it?
2. No, it takes seconds.